op-batcher/flags/flags.go

```diff
@@ -167,7 +167,7 @@ var (
 	}
 	ThrottleThresholdFlag = &cli.IntFlag{
 		Name:    "throttle-threshold",
-		Usage:   "The threshold on pending-blocks-bytes-current beyond which the batcher will instruct the block builder to start throttling transactions with larger DA demands. Zero disables throttling.",
+		Usage:   "The threshold on unsafe_da_bytes beyond which the batcher will start to throttle the block builder. Zero disables throttling.",
 		Value:   1_000_000,
 		EnvVars: prefixEnvVars("THROTTLE_THRESHOLD"),
 	}
```

Comment:
where does tx-throttling start at? we don't have a `throttle-always-tx-size` flag.

Reply:
Good callout, we currently treat transaction size like a step controller always:

https://github.com/ethereum-optimism/optimism/blob/cedca4164c900973db391ac39c1aae62a0c3168a/op-batcher/batcher/throttler/controller.go#L69-L82

Will follow up on another PR to address this.

Reply:
follow up sounds good!